content/docs/contribute/integrate-your-tech@es.md

```diff
@@ -0,0 +1,53 @@
+
+## 2. Usando la extensión amp-ad
+
+La extensión amp-ad está reservada para publicar anuncios de visualización en las páginas de AMP. Más de 90 proveedores de tecnología de anuncios han añadido soporte a AMP. Para empezar, lea la [descripción general del desarrollo](https://github.com/ampproject/amphtml/tree/master/ads#overview) o salte a las [instrucciones del desarrollador](https://github.com/ampproject/amphtml/tree/master/ads#developer-guidelines-for-a-pull-request) para agregar su soporte a la extensión amp-ad. Dependiendo de la tecnología de anuncios que su empresa proporcione, es posible que encuentre útiles estas [instrucciones de integración](https://github.com/ampproject/amphtml/blob/master/ads/_integration-guide.md).
+
+Hay más de 90 proveedores de anuncios que han añadido soporte para funciones publicitarias relacionadas, como amp-ad. A continuación se muestra una [solicitud de extracción](https://github.com/ampproject/amphtml/pull/2299) de muestra de la red publicitaria [Criteo](https://github.com/ampproject/amphtml/blob/master/ads/criteo.md).
+
+## 3. Uso de la extensión amp-call-tracking
+
+Si proporciona servicios de medición de seguimiento de llamadas, su caso de uso puede ser compatible con la nueva extensión [amp-call-tracking](https://github.com/ampproject/amphtml/blob/master/extensions/amp-call-tracking/amp-call-tracking.md). Esta extensión reemplaza dinámicamente un número de teléfono en un hipervínculo para habilitar el seguimiento de llamadas, ejecutando una petición de CORS para sustituir el número.
+
+Para obtener más información sobre cómo esta extensión podría funcionar para usted, consulte el sitio [AMP by Example](https://ampbyexample.com/components/amp-call-tracking/) o lea acerca de la [documentación](/docs/reference/components/amp-call-tracking).
+
+## 4. Agregar una nueva extensión o incrustar
+
+Si su caso de uso no se puede acomodar mediante amp-analytics, amp-pixel o amp-ad, reporte un [problema en GitHub](https://github.com/ampproject/amphtml/issues/new) para discutir opciones alternativas. Damos la bienvenida a las nuevas extensiones que pueden ser ampliamente utilizados por un número de diferentes empresas. Consulte la [sección de componentes ampliados](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#contributing-extended-components) para obtener más detalles.
+
+## 5. Usando el amp-iframe
+
+Se espera su uso solamente como último recurso. Si ninguno de las extensiones anteriores se ajustan a sus necesidades, podría utilizar la etiqueta genérica amp-iframe para permitir a los editores integrar su contenido, pero este enfoque viene con una serie de desventajas, para optimizar el rendimiento y la experiencia del usuario que puede leer [aquí](/docs/reference/components/amp-iframe#guideline:-prefer-specific-amp-components-to-amp-iframe).
```

Comment:
Please change the link to `/docs/reference/components/amp-iframe.html#guideline:-prefer-specific-amp-components-to-amp-iframe`